css/styles.css

```diff
@@ -457,13 +458,15 @@ html::-webkit-scrollbar-thumb {
   }
 
   .hero .content p {
-    width: 80%;
+    width: 85%;
+    text-align: center;
+    font-size: 1.8rem;
   }
 
   /* ******************** Program ***************************** */
   .program {
     width: 100%;
-    height: 82vh;
+    height: 100vh;
   }
 
   .program .row .card {
```

Comment:
Please ensure that the pages look almost identical to the original design. And please ensure there are no images or elements blocking the text in both screen sizes.
Your Implementation | Project Design

Your Implementation | Figma Design
:---:|:---:
  
![overlap](https://user-images.githubusercontent.com/10332499/152071480-ac7d5500-59f9-45f3-9709-5174b47b5c48.PNG)  | ![design](https://user-images.githubusercontent.com/10332499/152071501-dfb046de-9d8c-4a71-853f-e81614345b3a.PNG)